pkg/virt-controller/services/multus_annotations.go

```diff
@@ -33,21 +33,37 @@ type multusNetworkAnnotation struct {
 	Namespace     string `json:"namespace"`
 }
 
+type multusNetworkAnnotationPool struct {
+	pool []multusNetworkAnnotation
+}
+
+func (mnap *multusNetworkAnnotationPool) add(multusNetworkAnnotation multusNetworkAnnotation) {
+	mnap.pool = append(mnap.pool, multusNetworkAnnotation)
+}
+
+func (mnap multusNetworkAnnotationPool) isEmpty() bool {
+	return len(mnap.pool) == 0
+}
+
+func (mnap multusNetworkAnnotationPool) toString() (string, error) {
```

Comment:
Ahh, if you could have used `String`, it will fit the `Stringer` interface, which `fmt` can use then. But that error is in the way.

Reply:
I could return an empty string instead of an error if we really want to implement the `Stringer` interface. 

I don't think it's worth it, but I can compromise.

Reply:
No need from my part, I just shared my thought.